NOTES.md

```diff
@@ -66,3 +67,24 @@ A row in the actual data section of the database has a 26-byte row header. The m
 |  `0x9`   | 16   | Checksum (MD5?)        |
```

Comment:
In all files, this is the engine's checksum and not a file checksum. This needs to match the reading engine, otherwise the engine is unable to read the file. It doesn't matter for this project.